### PR TITLE
mon: add force-create-pg back

### PR DESCRIPTION
**Pull request by tchaikov**
and now it's "ceph osd force-create-pg'

Fixes: http://tracker.ceph.com/issues/20605
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
tested manually by inspecting the log file of mon and primary osd of the pg.

File: doc/man/8/ceph.rst [hunk 1]
```diff
@@ -1136,6 +1143,8 @@ Usage::
 
 	ceph pg force_create_pg <pgid>
 
+This command is DEPRECATED as of the Luminous release, and will be removed in a future release. Subcommand ``force-create-pg`` should instead be used.
+
```
Comment (liewegas) [inline doc/man/8/ceph.rst]: It's actually removed; it will only work during the upgrade.

**Comment by tchaikov** [comment]
@liewegas fixed and repushed

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-17_17:34:45-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**